guides/common/modules/con_working-with-lifecycle-environments.adoc

```diff
@@ -0,0 +1,17 @@
+[id="working-with-lifecycle-environments"]
+= Working with lifecycle environments
+
+{Project} divides application life cycles into lifecycle environments, which represent each stage of the application life cycle.
+Lifecycle environments are linked to from an environment path.
+To create linked lifecycle environments with the API, use the `prior_id` parameter.
+
+You can find the built-in API reference for lifecycle environments at `https://_{foreman-example-com}_/apidoc/v2/lifecycle_environments.html`.
+The API routes include `/katello/api/environments` and `/katello/api/organizations/:organization_id/environments`.
+
+proc_listing-lifecycle-environments
+
+proc_creating-linked-lifecycle-environments
+
+proc_updating-a-lifecycle-environment
+
+proc_deleting-a-lifecycle-environment.adoc
```

Comment:
This is looking strange...

Reply:
Yeah .... Removed and verified that it's included in the assembly.